src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -493,4 +494,11 @@ public sealed partial class JsonPropertyNameAttribute : System.Text.Json.Seriali
         public JsonPropertyNameAttribute(string name) { }
         public string Name { get { throw null; } }
     }
+    public sealed partial class JsonStringEnumConverter : System.Text.Json.Serialization.JsonConverterFactory
+    {
+        public JsonStringEnumConverter() { }
+        public JsonStringEnumConverter(System.Text.Json.JsonNamingPolicy namingPolicy = null, bool allowIntegerValues = true) { }
+        public override bool CanConvert(System.Type type) { throw null; }
```

Comment:
@JeremyKuhne, I believe the parameter name needs to be `typeToConvert` to match the base class, and not `type`.